node/src/command.rs

```diff
@@ -249,54 +247,52 @@ pub fn run() -> Result<()> {
         None => {
-            let runner = cli.create_runner(&*cli.run)?;
-
-            runner.run_node_until_exit(|config| {
-                async move {
-                    // TODO
-                    let key = sp_core::Pair::generate().0;
-
-                    let extension = chain_spec::Extensions::try_get(&*config.chain_spec);
-                    let relay_chain_id = extension.map(|e| e.relay_chain.clone());
-                    let para_id = extension.map(|e| e.para_id);
-
-                    let polkadot_cli = RelayChainCli::new(
-                        config.base_path.as_ref().map(|x| x.path().join("polkadot")),
-                        relay_chain_id,
-                        [RelayChainCli::executable_name().to_string()].iter().chain(cli.relaychain_args.iter()),
-                    );
-
-                    let id = ParaId::from(cli.run.parachain_id.or(para_id).unwrap_or(200));
-
-                    let parachain_account =
-                        AccountIdConversion::<polkadot_primitives::v0::AccountId>::into_account(&id);
-
-                    let block: Block = generate_genesis_block(&config.chain_spec).map_err(|e| format!("{:?}", e))?;
-                    let genesis_state = format!("0x{:?}", HexDisplay::from(&block.header().encode()));
-
-                    let task_executor = config.task_executor.clone();
-                    let polkadot_config =
-                        SubstrateCli::create_configuration(&polkadot_cli, &polkadot_cli, task_executor)
-                            .map_err(|err| format!("Relay chain argument error: {}", err))?;
-                    let collator = cli.run.base.validator || cli.collator;
-
-                    info!("Parachain id: {:?}", id);
-                    info!("Parachain Account: {}", parachain_account);
-                    info!("Parachain genesis state: {}", genesis_state);
-                    info!(
-                        "Is collating: {}",
-                        if collator {
-                            "yes"
-                        } else {
-                            "no"
-                        }
-                    );
-
-                    crate::service::start_node(config, key, polkadot_config, id, collator)
-                        .await
-                        .map(|r| r.0)
-                        .map_err(Into::into)
-                }
+            let runner = cli.create_runner(&cli.run.normalize())?;
+
+            runner.run_node_until_exit(|config| async move {
+                let para_id =
+                    chain_spec::Extensions::try_get(&*config.chain_spec).map(|e| e.para_id);
+
+                let polkadot_cli = RelayChainCli::new(
+                    &config,
+                    [RelayChainCli::executable_name().to_string()]
+                        .iter()
+                        .chain(cli.relaychain_args.iter()),
+                );
+
+                let id = ParaId::from(cli.run.parachain_id.or(para_id).unwrap_or(2000));
```

Comment:
could we defined this parachain id in one place and reuse (see other comment) instead of hardcoding and repeating everywhere